### PR TITLE
Add auto_update feature for neovim

### PR DESCRIPTION
**Pull request by AkiraHasegawa1997**
Hi,
I'm really sorry to send the same pull request twice.
Since I forgot to make a branch before sending the pull request, I tried to fix it but accidentally closed the previous pull request.

This commits allows to use auto_update feature for Neovim users.

- To save a job id that is made by `termopen`, I made a new variable `s:preview_job_id`.

- To close the buffer of a terminated terminal, I added the following lines:
```
  if has('nvim')
    autocmd TermClose * exe "bd! " . expand('<abuf>')
  endif
```

### CODE REVIEW

**Comment by AkiraHasegawa1997** [comment]
When I resend the pull request, a critical bug had been introduced, so I fixed it by force push.

**Comment by AkiraHasegawa1997** [comment]
Sorry, I did a force-push again. 
I found a bug that when we select `glow` instead of `mdr`, it cannot work correctly due to the autocmd modification. 
So I changed that to do `execute bd` explicitly instead of autocmd.

File: autoload/preview_markdown.vim [hunk 1]
```diff
@@ -22,19 +23,29 @@ function! s:remove_tmp(tmp, channel, msg) abort
...
-    while job_status(s:jobid) is# 'run'
-      if c > 5
-        call s:echo_err('stop job is timeout')
-        return
+  if has('nvim')
+    let s:jobid = s:preview_job_id
+    if s:jobid isnot# v:null
+      call jobstop(s:jobid)
+      if bufexists(s:preview_buf_nr)
+          execute "bd! " . s:preview_buf_nr
```
Comment (skanehira) [inline autoload/preview_markdown.vim]: I think we should use `bw! ` to wipe out existed buffer

**Review by skanehira — CHANGES_REQUESTED** [review]
Thank you :)
I left a comment

**Comment by AkiraHasegawa1997** [comment]
Thank you for the very helpful comment. I fixed it to use `bw!` instead of `bd!`.

**Comment by skanehira** [comment]
@AkiraHasegawa1997 Thank you
At last one, could you resolve conflicting?
I'll merge this PR when conflict be resolved

**Comment by skanehira** [comment]
@AkiraHasegawa1997 thank you :)

**Pull request merged**